### DISCUSSION TITLE
Falcon Custom Sponsorship

### DISCUSSION THREAD

**Discussion by AdrianMachado** [falconry/falcon#2220] [General]
Hey Falcon Team,

My company is interested in supporting and sponsoring Falcon. We are looking for a sponsorship a bit our of the regular terms. We've found that homepage banners perform best for us (ex. https://rapidocweb.com/ and https://www.jsonschema.net/). Would you be open to having a sponsored banner on your homepage for an increased sponsorship fee?

**Comment by vytas7** [comment]
Hi @AdrianMachado!
And thanks for your interest in sponsoring Falcon! It is normal to show logotypes and sponsors, there is nothing unusual in that, as you can see, we have logotypes on https://falconframework.org/. (And we used to have on ReadTheDocs as well, although something broke there with the way we include that list.) We will probably review the list on the website soon too, because not all the listed sponsors are active.

Currently the [Platinum Membership](https://opencollective.com/falcon) starts at 250$/month which also gives a large logotype, but as I understand, that is not enough according to your marketing vision, and you are after an even more intrusive full width banner on the header of the website? We need to discuss this within the maintainer team if that is the case.

**Reply by AdrianMachado** [reply]
Hey @vytas7 - thanks for the quick reply. Here's what we are looking for

In terms of what we are looking to achieve:
1. Drive mutual growth (ex. Zuplo customer growth and Falcon developer growth). We measure growth in terms of traffic and signups that we can attribute to Falcon. I believe this is mutually beneficial, as many Falcon devs likely want a quick solution to securing and scaling their APIs, whereas many Zuplo customers look for good frameworks to build their APIs that integrate well with Zuplo.
2. Support the OpenSource API development community - we're all one big happy family :) 

We were thinking of the following:

We Collaborate on
1. A walkthrough on building an API using Falcon and deploying on Zuplo. I was thinking we make a simple project with a few endpoints in Falcon, show how to export the OpenAPI (if Falcon supports this), deploy to somewhere like [Vercel](https://vercel.com/docs/functions/runtimes/python), import the OpenAPI into Zuplo (or using the UI to create a proxy if there is no OpenAPI file), and start proxying the Falcon endpoints - adding Rate limiting and API key authentication

You Provide
1. Either a homepage banner ([example](https://www.jsonschema.net/)) or a prominent sponsor feature on Falcon's homepage. We found that these drive the most conversions, which allows a small startup like us to continue to fund our sponsorships
2. Sponsor section in the README
3. Links to the Falcon/Zuplo walkthrough in your docs

We Provide
1. Links to the Falcon/Zuplo walkthrough in our docs
2. We will produce a walkthrough video for social media on using Falcon and Zuplo
3. Monthly monetary contribution (amount to be negotiated)

Let me know if you would like me to clarify any of the above points.

**Comment by vytas7** [comment]
Hi again @AdrianMachado!
I see, it does look like a sound marketing plan which could potentially benefit your business while also bringing more resources into Falcon's development.

At the same time some of the proposed action points spontaneously feel a bit intrusive, so we would need to go through the details, and reach a consensus within our maintainer team.

Would you be available for a private call/further discussions?

**Reply by AdrianMachado** [reply]
@vytas7  You can grab time with me for a call here: https://calendly.com/adrian-zuplo/30min

**Reply by AdrianMachado** [reply]
It would be great if you could have your monthly traffic/engagement numbers for both your repo and website on-hand to inform the discussion. Looking forward to chatting :)
